.github/workflows/propagate_to_osbuild.yml

```diff
@@ -0,0 +1,64 @@
+name: Propagate update to OSBuild
```

Comment:
I have basically the same concerns about this whole file as in https://github.com/osbuild/osbuild/pull/1124.

Reply:
I think I've addressed them correctly :)